### ISSUE TITLE
Accessibility Violation - aria-required-parent for role="treeitem"

### ISSUE THREAD

**Issue by sparekh0730** [brimdata/react-arborist#197]
The row-container component defines an aria role of "treeitem", but does not give the tree-container an aria role of "tree". This is in violation of accessibility guidelines https://www.w3.org/TR/wai-aria/#treeitem.

<img width="655" alt="Screenshot 2023-12-04 at 2 47 10 PM" src="https://github.com/brimdata/react-arborist/assets/24374917/b1e7261e-ce91-4f40-818d-1ed24093673a">

Could you please define an aria role of "tree" for the tree container?

**Comment by jameskerr** [comment]
Just saw your PR about this. Thank you!
